src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,171 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private User bobFirst;
+    private User bobSecond;
+    private User alice;
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        bobFirst = new User();
+        bobFirst.setLogin("Bob_First");
+        bobFirst.setAge(32);
+        bobFirst.setPassword("123456");
+        bobSecond = new User();
+        bobSecond.setLogin("Bob_Second");
+        bobSecond.setAge(31);
+        bobSecond.setPassword("123456");
+        alice = new User();
+    }
+
+    @Test
+    void registrationNormal_OK() {
```

Comment:
```suggestion
    void register_ValidUser_OK() {
```
test names should follow pattern
`<method name>_<description>_<expected behavior>`